File: src/main/webapp/jsLibrary/js/service/addressService.js
app.service("addressService", function ($http) {

    this.findAreas = function (citiesId) {
        return $http.get("../address/findAreas?citiesId=" + citiesId)
    }
    this.findCities = function (provincesId) {
        return $http.get("../address/findCities?provincesId=" + provincesId);
    }
    this.findProvince = function (provincesId) {
        return $http.get("../address/findProvince?provincesId=" + provincesId);
    }
    this.findProvincesAll = function () {
        return $http.get("../address/findProvincesAll");
    }
    this.findAreasAll = function () {
        return $http.get("../address/findAreasAll");
    }
    this.findCitieAll = function () {
        return $http.get("../address/findCitieAll");
    }

})

File: src/main/java/com/meeting/m2/service/AddressService.java
package com.meeting.m2.service;

import com.meeting.m2.pojo.Areas;
import com.meeting.m2.pojo.Cities;
import com.meeting.m2.pojo.Meeting;
import com.meeting.m2.pojo.Provinces;

import java.util.List;

public interface AddressService {


    List<Areas> findAreas(int citiesId);

    List<Cities> findCities(int provincesId);

    Provinces findProvinces(int provincesId);

    List<Provinces> findProvincesAll();

    List<Areas> findAreasAll();

    List<Cities> findCitieAll();
}
